core/src/labels/textLabel.cpp

```diff
@@ -123,4 +123,8 @@ void TextLabels::setQuads(std::vector<GlyphQuad>& _quads) {
     style.context()->lockAtlas(m_atlasRefs);
 }
 
+size_t TextLabels::bufferSize() const {
+    return style.meshBufferSize();
+}
+
```

Comment:
Not sure, but this should consistently return the actual size, but maybe the cache shouldn't consider it since it's a per frame usage?